### PR TITLE
release: v2.19.0

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,39 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.19.0](https://github.com/serverless/serverless/compare/v2.18.0...v2.19.0) (2021-01-15)
+
+### Features
+
+- Seclude main error handler to standalone util ([#8726](https://github.com/serverless/serverless/pull/8726)) ([847fa34](https://github.com/serverless/serverless/commit/847fa3412d221c2ff98ab0cd9165bfc193c8a224)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:9]: Let's move it to _Maintainance_ section. By mistake I made it a `feat`, it's rather `refactor`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,39 @@
...
+- **Variables:**
+  - Introduce unresolvedVariablesNotificationMode ([#8710](https://github.com/serverless/serverless/issues/8710)) ([33cffc3](https://github.com/serverless/serverless/commit/33cffc3509255663c9ab94f3cd38f115d71bd1d2)) ([Gareth Jones](https://github.com/G-Rath))
+  - Add support for Terraform state file parsing ([#8755](https://github.com/serverless/serverless/issues/8755)) ([461a396](https://github.com/serverless/serverless/commit/461a3965a52eb9707121700608dc8bdbafc367d1)) ([Brian Dwyer](https://github.com/bdwyertech))
+
+### Bug Fixes
+
+- **AWS CloudFront:** Fix deprecations visibility ([#8759](https://github.com/serverless/serverless/pull/8759)) ([6c67cd7](https://github.com/serverless/serverless/commit/6c67cd7f074ef27c9410f29b368dc7e87b5b6e2d)) ([Mariusz Nowak](https://github.com/medikoo))
+- **CLI:** Remove bluebird config ([#8732](https://github.com/serverless/serverless/pull/8732)) ([1019e60](https://github.com/serverless/serverless/commit/1019e608762c45d1c868fce15224aca4c4e5a56c)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:17]: Let's remove this item (this was a fix to `master` but it's not a fix against previous version, it's more a complement to #8726 )

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,39 @@
...
+- **Config Schema:** Revert to ajv v6 ([#8762](https://github.com/serverless/serverless/issues/8762)) ([d1c6568](https://github.com/serverless/serverless/commit/d1c656838f5d19dd2b1d214c30ea2f292915a5b2)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **Packaging:** Consider absolute artifact paths ([#8325](https://github.com/serverless/serverless/issues/8325)) ([#8315](https://github.com/serverless/serverless/issues/8315)) ([bcbbd47](https://github.com/serverless/serverless/commit/bcbbd47fa09b7d99d7f8da3f11150215d1203bba)) ([Robert Bragg](https://github.com/rib) & [Piotr Grzesik](https://github.com/pgrzesik))
+
+### Maintenance Improvements
+
+- Abstract resolution of deployment role ([#8751](https://github.com/serverless/serverless/issues/8751)) ([4afdb83](https://github.com/serverless/serverless/commit/4afdb8314b5c4718e73de733e3c4b30ae62382ba)) ([Dmitry Shirokov](https://github.com/runk))
+- Cleanup mergeIamTemplates module ([#8736](https://github.com/serverless/serverless/issues/8736)) ([77e1a6a](https://github.com/serverless/serverless/commit/77e1a6a30246f94fcdf8ae26ca2cb8617aa1db2b)) ([Dmitry Shirokov](https://github.com/runk))
```
Comment (medikoo) [inline CHANGELOG.md:24]: Let's maybe format function names with back quotes

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,39 @@
...
+- Improve error handling scope ([#8726](https://github.com/serverless/serverless/pull/8726)) ([49aabdf](https://github.com/serverless/serverless/commit/49aabdf13d2ee74380ec2d21f57ffde494a9bf9d)) ([Mariusz Nowak](https://github.com/medikoo))
+- Improve var naming ([#8726](https://github.com/serverless/serverless/pull/8726)) ([7136850](https://github.com/serverless/serverless/commit/71368507e4df5b5582bafb3db9dc8a2e89a8ed8a)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:26]: I think we may skip that one (very small obviously not affecting refactors I would not list)

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,39 @@
...
+- Reconfigure onExitPromise setup ([#8726](https://github.com/serverless/serverless/pull/8726)) ([22a03ce](https://github.com/serverless/serverless/commit/22a03ce0d7b1581747b121f862d0818f04120958)) ([Mariusz Nowak](https://github.com/medikoo))
+- Refactor init part to async/await ([#8726](https://github.com/serverless/serverless/pull/8726)) ([763fc34](https://github.com/serverless/serverless/commit/763fc342f7d82a86db00cbccab2f847d943d3e9e)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:28]: I would probably skip that one too (it was shadowed later by `Seclude main error handler to standalone util` one), and users may confirm in PR all scope of changes

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,39 @@
...
+- Refactor Serverless.run to async ([#8749](https://github.com/serverless/serverless/pull/8749)) ([30015ea](https://github.com/serverless/serverless/commit/30015eafd2fb9d2e82d8f34ee8f10c1fb4e536a0)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- Reorder init setup ([#8726](https://github.com/serverless/serverless/pull/8726)) ([7548e1d](https://github.com/serverless/serverless/commit/7548e1d847f70127384400c027de3b6386fde698)) ([Mariusz Nowak](https://github.com/medikoo))
```
Comment (medikoo) [inline CHANGELOG.md:30]: Let's skip that one

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,39 @@
...
+- Seclude ServerlessError into lib/serverless-error.js ([#8743](https://github.com/serverless/serverless/pull/8743)) ([87790e5](https://github.com/serverless/serverless/commit/87790e50bd9c178aefd4f2ad8793c9c56fb8eb49)) ([Mariusz Nowak](https://github.com/medikoo))
+- Typos in schema ([#8735](https://github.com/serverless/serverless/issues/8735)) ([2b7568a](https://github.com/serverless/serverless/commit/2b7568a960c88dda8ab2bbe1b6c8dd238fa78a51)) ([Dmitry Shirokov](https://github.com/runk))
+- **CLI:** Seclude version output functionality out of CLI class ([#8741](https://github.com/serverless/serverless/pull/8741)) ([b61621a](https://github.com/serverless/serverless/commit/b61621adebb7eb33fd080db3fff13d7e9a32d99b)) ([Mariusz Nowak](https://github.com/medikoo))
+- **Templates:** Fix typo in package.json for template aws-nodejs-typescript ([#8754](https://github.com/serverless/serverless/issues/8754)) ([37398d0](https://github.com/serverless/serverless/commit/37398d06c582b1676c2aaa32708cfd515baf65b9)) ([Alexandre de Boutray](https://github.com/aldebout))
```
Comment (medikoo) [inline CHANGELOG.md:36]: Let's put updates related to _Templates_ to individual `Templates` section (there's somewhat independent of Framework logic, so I would not mix it in)

**Review by medikoo — CHANGES_REQUESTED** [review]
Thanks @pgrzesik I have just few improvement suggestions to changelog, so it's more substantial, and we should be good to go

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,38 @@
...
+- Cleanup `mergeIamTemplates` module ([#8736](https://github.com/serverless/serverless/issues/8736)) ([77e1a6a](https://github.com/serverless/serverless/commit/77e1a6a30246f94fcdf8ae26ca2cb8617aa1db2b)) ([Dmitry Shirokov](https://github.com/runk))
+- Improve error handling scope ([#8726](https://github.com/serverless/serverless/pull/8726)) ([49aabdf](https://github.com/serverless/serverless/commit/49aabdf13d2ee74380ec2d21f57ffde494a9bf9d)) ([Mariusz Nowak](https://github.com/medikoo))
+- Reconfigure onExitPromise setup ([#8726](https://github.com/serverless/serverless/pull/8726)) ([22a03ce](https://github.com/serverless/serverless/commit/22a03ce0d7b1581747b121f862d0818f04120958)) ([Mariusz Nowak](https://github.com/medikoo))
+- Refactor Serverless.run to async ([#8749](https://github.com/serverless/serverless/pull/8749)) ([30015ea](https://github.com/serverless/serverless/commit/30015eafd2fb9d2e82d8f34ee8f10c1fb4e536a0)) ([Piotr Grzesik](https://github.com/pgrzesik))
```
Comment (medikoo) [inline CHANGELOG.md:25]: It'll be nice to also backquote above two

File: CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,38 @@
...
+- Seclude `cli/resolve-local-serverless-path` util ([#8726](https://github.com/serverless/serverless/pull/8726)) ([9d78348](https://github.com/serverless/serverless/commit/9d783482895d82a1bfdb627c4cc0debb32123d56)) ([Mariusz Nowak](https://github.com/medikoo))
+- Seclude `ensureExists` util ([#8744](https://github.com/serverless/serverless/pull/8744)) ([c3f59e4](https://github.com/serverless/serverless/commit/c3f59e4d785145c2e1ba7c1324f3afedba482479)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- Seclude `ServerlessError` into `lib/serverless-error.js` ([#8743](https://github.com/serverless/serverless/pull/8743)) ([87790e5](https://github.com/serverless/serverless/commit/87790e50bd9c178aefd4f2ad8793c9c56fb8eb49)) ([Mariusz Nowak](https://github.com/medikoo))
+- Typos in schema ([#8735](https://github.com/serverless/serverless/issues/8735)) ([2b7568a](https://github.com/serverless/serverless/commit/2b7568a960c88dda8ab2bbe1b6c8dd238fa78a51)) ([Dmitry Shirokov](https://github.com/runk))
```
Comment (medikoo) [inline CHANGELOG.md:29]: This should be moved to _fixes_, as that affects our validation functionality

**Review by medikoo — CHANGES_REQUESTED** [review]
Sorry two more picks and we should be good to go :)

File: CHANGELOG.md (hunk 9 above)
Comment (pgrzesik) [reply CHANGELOG.md:29]: I believe that's not the case as the typos were made in definitions, so it doesn't affect any external functionality

File: CHANGELOG.md (hunk 8 above)
Comment (pgrzesik) [reply CHANGELOG.md]: done :+1:

File: CHANGELOG.md (hunk 9 above)
Comment (medikoo) [reply CHANGELOG.md:29]: Indeed, I looked again, you're right. Sorry for confusion!

**Review by medikoo — APPROVED** [review]
👍

**Pull request merged**